src/cassandra.go

```diff
@@ -104,14 +104,25 @@ func main() {
 
 // runMetricCollection will perform the metrics collection.
 func runMetricCollection(i *integration.Integration, jmxClient *gojmx.Client) error {
+	definitions := GetDefinitions()
+
+	config, err := LoadConfig()
+	if err != nil {
+		// Not a fatal error.
+		log.Debug(
+			"no extra configuration loaded: %v", err)
```

Comment:
why not a fatal error if the config is wrong? isn't there the risk of hiding issues?

Reply:
this is for extra configuration and has no impact in the ability to collect metrics. This is why I don't think is a critical error. But if you think it would be better to fail, we can change it

Reply:
yeah I think that everytime the user configure something in a bad way it should be as clear as possible